core/vnl/algo/tests/test_symmetric_eigensystem.cxx

```diff
@@ -92,63 +92,63 @@ test_symmetric_eigensystem()
   { // compare speed and values of specialised 3x3 version with nxn version
     constexpr unsigned n = 20000;
 
+    // Generate a random system with seeded value to known value
```

Comment:
I did not actually review this refactoring commit. Possibly remove `# Please enter the commit message ...` from commit message.

Reply:
Done.